jetstream/cli.py

```diff
@@ -613,6 +630,19 @@ def dataset_id_option(default="mozanalysis"):
     multiple=True,
 )
 
+image_option = click.option(
+    "--image",
+    help="Name of the docker image to use in Argo.",
+    default="jetstream",
+)
+
+image_version_option = click.option(
+    "--image_version",
+    "--image-version",
```

Comment:
Suggestion to include the docker terminology as well:
```suggestion
    "--image-version",
    "--image-tag",
```

Reply:
mh, I guess this isn't technically referring to the tag, which would be something we'd need to assign (or `latest` which gets assigned automatically and is more of a special case here).

Reply:
Yea, fair enough, between docker assigning a hash as default tag and the `latest` default, I got myself mixed up here.